pages/ugm2023.md

```diff
@@ -70,6 +70,25 @@ There are many other hotel and guest house options accesible to and from the ven
 
 Citizens of several countries may travel to Portugal to attend a meeting without obtaining a visa. Please visit the [Portugal Ministry of Foreign Affairs website][mne.gov.pt] for more information about travel visas. Confirmed attendees requiring a letter clarifying the purpose of travel should [contact the MDAnalysis project manager][email]. 
 
+## Health and Safety
+We are committed to providing a safe, comfortable, and healthy environment for the MDAnalysis UGM and have looked to local and international entities and other organizations in the open source community to develop onsite safety measures for UGM attendees.
+
+### Vaccine Policy
+MDAnalysis strongly recommends that UGM attendees remain current (i.e., receiving all recommended primary and booster doses recommended by your local government) with their COVID-19 vaccinations as a personal protective measure. 
+
+### Testing
+A COVID-19 test result is not required to determine whether you can attend the UGM. A limited number of rapid self-tests will be made available at check-in for those who would like to test themselves for COVID-19 during the meeting.
+
+*Testing Positive for COVID-19*
+
+Individuals who are COVID-positive should not attend the UGM. If you receive a positive COVID-19 test result during the UGM, please [notify the MDAnalysis UGM organizing committee][ugm_email] and do not return to the event while you are still COVID-positive.
+
+### Face Coverings
+Face coverings are not required, but are welcomed. We view face coverings as a personal choice for protecting oneself and will support attendees in their choice to wear or not to wear a face covering.
```

Comment:
I'd remove "for protecting oneself" because once you're saying "personal choice" it's their choice for whatever reason — for instance, I might also make a choice to protect others _and_ myself. 

Simply speaking, the less we are saying the better.

Reply:
```suggestion
Face coverings are not required, but are welcomed. We view face coverings as a personal choice and will support attendees in their choice to wear or not to wear a face covering.
```